force-app/main/default/classes/IBMSpeechToTextV1.cls

```diff
@@ -798,6 +802,39 @@ public class IBMSpeechToTextV1 extends IBMWatsonService {
     if (trainLanguageModelOptions.customizationWeight() != null) {
       builder.query('customization_weight', String.valueOf(trainLanguageModelOptions.customizationWeight()));
     }
+    if (trainLanguageModelOptions.strict() != null) {
```

Comment:
We need to remove the `strict` param until Jeff and the Speech team sort out the response.